<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preprocess_dialog</class>
 <widget class="QDialog" name="preprocess_dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preprocess TIFF images</string>
  </property>
  <widget class="QPushButton" name="run_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Module 1 - Preprocess TIFF images</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="log_textbox_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Running log&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="log_textbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="source_group_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Source directory</string>
   </property>
   <widget class="QLineEdit" name="source_dir_textbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="select_source_dir_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>161</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Select directory</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="output_group_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Output directory</string>
   </property>
   <widget class="QLineEdit" name="output_dir_textbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="select_output_directory">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>161</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Select output directory</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="version_label">
   <property name="geometry">
    <rect>
     <x>496</x>
     <y>460</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>version label</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="app_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mouse Brain Alignment Tool</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
